src/ray/protobuf/gcs_service.proto

```diff
@@ -92,6 +92,19 @@ message GetAllActorInfoReply {
   repeated ActorTableData actor_table_data = 2;
 }
 
+message KillActorViaGcsRequest {
```

Comment:
core_worker.proto has `message KillActorRequest`. To avoid duplicate names, we name it `KillActorViaGcsRequest`. Let me know if you have a better name.

Reply:
Can you add a comment in this message what's the difference from a regular killActor?

Reply:
Ok, thanks.